<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotwindow</class>
 <widget class="QMainWindow" name="plotwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLASS graph</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="modelruntree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>150</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::VLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="plotLayoutGrid" rowminimumheight="20,30,30,30,30,20">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="layout_xmin">
             <item>
              <widget class="QLabel" name="xminLabel">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>X-min:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="xminInput">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="layout_xmax">
             <item>
              <widget class="QLabel" name="xmaxLabel">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>X-max:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="xmaxInput">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="layout_ymin">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="yminLabel">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Y-min:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="yminInput">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="layout_ymax">
             <item>
              <widget class="QLabel" name="ymaxLabel">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Y-max:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ymaxInput">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="sw_scatterplot">
             <property name="text">
              <string>Scatter plot</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="autoscaleaxis">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Auto scale axis</string>
             </property>
             <property name="iconSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="centralLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="dummy_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="AdvancedDock">
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>146</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Plot variables</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="AdvancedDockContent">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="advancedplottree">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_savepng"/>
    <addaction name="separator"/>
    <addaction name="menu_exit"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="view_advancedplotsettings"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="view_menu"/>
  </widget>
  <action name="menu_modelruns">
   <property name="text">
    <string>Modelruns</string>
   </property>
  </action>
  <action name="menu_basicplotting">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Basic plotting</string>
   </property>
  </action>
  <action name="menu_advancedplotting">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Advanced plotting</string>
   </property>
  </action>
  <action name="menu_axis">
   <property name="text">
    <string>Axis settings</string>
   </property>
  </action>
  <action name="menu_saveimage">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="menu_savepng">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="menu_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menu_modelrun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Model runs</string>
   </property>
  </action>
  <action name="menu_plotsettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot settings</string>
   </property>
  </action>
  <action name="menu_basicplot">
   <property name="text">
    <string>Basic plotting</string>
   </property>
  </action>
  <action name="menu_advancedplot">
   <property name="text">
    <string>Advanced plotting</string>
   </property>
  </action>
  <action name="actionAdvanced_plot_settings">
   <property name="text">
    <string>Advanced-plot settings</string>
   </property>
  </action>
  <action name="actionAdvanced_plot_settings_2">
   <property name="text">
    <string>Advanced-plot settings</string>
   </property>
  </action>
  <action name="view_basicmode">
   <property name="text">
    <string>Basic mode</string>
   </property>
  </action>
  <action name="view_advancedmode">
   <property name="text">
    <string>Advanced mode</string>
   </property>
  </action>
  <action name="view_basicplotsettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic-plot settings</string>
   </property>
  </action>
  <action name="view_advancedplotsettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced-plot settings</string>
   </property>
  </action>
  <action name="view_axissettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Axis settings</string>
   </property>
  </action>
  <action name="view_modelruns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Model runs</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>advancedplottree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
